projects/rero/ng-core/src/lib/translate/date-translate-pipe.ts

```diff
@@ -34,16 +34,14 @@ export class DateTranslatePipe extends DatePipe implements PipeTransform {
     super(_translateService.currentLanguage);
   }
 
-  transform(
-    value: any,
-    format = 'mediumDate',
-    timezone?: string,
-    locale?: string
-  ): string {
+  transform(value: Date | string | number, format?: string, timezone?: string, locale?: string): string | null;
+  transform(value: null | undefined, format?: string, timezone?: string, locale?: string): null;
+  transform(value: Date | string | number | null | undefined, format?: string, timezone?: string, locale?: string): string | null
+  {
     if (!locale) {
       locale = this._translateService.currentLanguage;
     }
 
-    return super.transform(value, format, timezone, locale);
+    return super.transform(value);
```

Comment:
Why removing `format, timezone, locale` from parameters ? There seem still valid in Angular11

Reply:
+1